%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Introduction \stephanieComment{Stephanie's Section!}}

Biological mechanisms are complex and diverse, and are often characterized by a
slow accumulation of qualitative data \nathanComment{maybe include 'their
understanding' is often characterized by ... }. Our understanding of biological
systems can be expanded greatly by applying quantitative, predictive models to
pervasive phenomena that, in the absence of models, exist as a collection of
loosely-associated facts \talComment{How about: ``The recent rise of quantitative, mechanistic models has greatly bolstered our understanding of fundamental cellular processes such as...\textit{(list some awesome examples with references)}.'' Then maybe a sentence talking about how such disparate phenomena are actually related by very similar underlying mechanisms (as a transition towards the next sentence)}. Allosteric conformational change is an example of
such a pervasive phenomenon. Allosteric proteins switch between conformational
states in response to ligand binding. Perhaps most famously characterized in
hemoglobin, allostery is also very common among transcription factors that
switch between ``active'' and ``inactive'' states with significantly different
regulatory capabilities. \talComment{Could use a transition sentence here} Here, we use quantitative models to describe
transcriptional regulation through allosteric induction, and show that such
models can be used to predict the behavior of novel systems.\nathanComment{
With the first few lines, I worry that it'll come off as too belittling to
non-quantitative approaches}\griffinComment{I usually dislike the idea that anything not quantitative is useless. I agree that we should soften it. We should emphasize that if we can predict the behavior of the system from a very general point of view, we understand.}

As the theoretical basis for our models, we turn to what may be the most
powerful \talComment{interesting word choice...} model of allostery, the Monod-Wyman-Changeux (MWC) model
\cite{MONOD1965}. \griffinComment{We need to standardize the cite keys} Briefly,
the MWC model stipulates that a symmetrical allosteric protein is capable of
\sout{accessing at least two states} \talComment{accessing two distinct conformations}, a ``tense'' and ``relaxed'' state (or, as we will
refer to them here, an ``active'' and ``inactive'' state) \talComment{Could italicize rather than use parentheses}. These states exist in
equilibrium within a system, and a protein may switch between states at any
time \manuelComment{We should avoid using the word time since it is forbidden in the theoretical framework we are using}. Both states are capable of binding to a ligand \manuelComment{here should be plural}, though the protein's
affinity for the ligand will be affected significantly by the protein's
allosteric conformation. Given a high enough concentration of ligand, it becomes
energetically favorable for a protein to spend the significant majority of its
time in the state that favors ligand binding, thus shifting the overall protein
population towards the ligand-favoring state. We have previously discussed the
statistical mechanics of MWC models \cite{Marzen2013}, and we use such a
statistical mechanical framework for our model of transcriptional induction.
\nathanComment{we say statistical mechanics a lot. Maybe remove the second one
	in this sentence.} \talComment{I think the last 4 sentences can be tightened up}

\talComment{Abrupt transition. This seems like the place to mention that we are going to focus on induction of the Lac repressor in \textit{Escherichia coli} and its effects of transcriptional regulation. The next sentence here may be too much detail for the intro, but we should definitely keep what has already been done.} \manuelComment{I agree. The first time I read it I thought this was too many details for the intro.} To provide a straightforward context for exploring allosteric induction
\textit{in vivo}, we use a chromosomally-integrated simple repression construct
in which a single repressor binding site blocks access to an RNAP polymerase
binding site. A YFP reporter gene downstream of this construct allows us to
measure the fold-change caused by repressor binding. As our model repressor, we
use the Lac repressor (LacI), which has long been recognized as a rich model
for understanding the nature of allostery within the MWC framework
\cite{Daly1986, Dunaway1980, Meyer2013, Daber2007, Daber2009,
	Muller-Hartmann1996, OGorman1980, Sharp2011, Taraban2008, Wilson2007}.
Additionally, in previous studies we have carefully quantified LacI repression
as a function of repressor copy number and promoter affinity \cite{Garcia2011},
as well as gene copy number and competitor binding sites \cite{Weinert2014}. All
of these tunable parameters have been well characterized by statistical
mechanical models \cite{Bintu2005a, Bintu2005b}, allowing us to go beyond merely
characterizing known data and enable us to predict \textit{a priori} the
behavior of novel transcriptional architectures that can then be tested against
experimental data. \griffinComment{Maybe we should leave this section for the
	results/methods. We could just emphasize that we are testing this theory in an
	experimental system where we have quantitative control of a number of
	biophysical parameters.}

This work aims to extend our statistical mechanical models to include
allosteric induction by examining the
feedback mechanism by which the \textit{lac} system responds to changing levels
of lactose \cite{JACOB1961} \manuelComment{There is not a single feedback component on our system. This sentence is very misleading}. We extend the previous thermodynamic models for
simple repression to include the effects of an inducer. This introduces three
new parameters into our model: the binding affinity of inducer to LacI in the
``active'' or DNA-binding conformation, the binding affinity of inducer to LacI
in the ``inactive'' or non-DNA-binding conformation, and the free energy
difference between these two conformations \talComment{I think this sentence can be moved into the previous paragraph. I think the next sentence should be removed, since our emphasis is not on fitting, but on our parameter-free predictive power}. We discuss how these parameters can
be inferred by fitting our model to repression data acquired under varying
concentrations of IPTG \nathanComment{Since we tend to talk about fold-change,
or fold-change in expression, maybe used 'expression' instead of 'repression'
data}. With these new parameters in hand, we quantitatively
predict the induction profile for other strains in which operator affinity and
LacI copy number have been varied. We analyze more than 20 such mutants and show
that in each case, the theoretical predictions match the experimental
measurements \talComment{This is the heart of our new contributions, and I think this should be emphasized much more and moved further back. This is the advertisement for why people should care about this paper}. In addition, we show that the mechanisms governing fold-change in
each mutant are systematically equivalent by identifying the ``natural
parameter'' governing IPTG induction; when plotted against this natural
parameter, the fold-change values for all of the mutants fall on the same curve.
In this way, we demonstrate the power of quantitative modeling as a tool for
achieving deep understanding of biological systems. \nathanComment{Switches from
strain to mutant half way through. I'd avoid using mutant} \manuelComment{I feel that the last part of the intro needs to emphasize more on what are our contributions. Give away the punchline in a way that if someone only reads the intro they can get the general idea of what we showed.}
