\pagebreak
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Experimental Design.} \nathanComment{One other thought would be to follow Hernan's  PNAS approach: Have a Theory and Experimental Design instead of Results above; then have Results section here.}

%\stephanieComment{General comments on this section: Mainly, I think we could expand upon the prediction of expression and provide more justification for data	collapse. We might want to refer to some quantitative comparison between the	quality of the fits from a single strain vs. the fits from all the data sets, and comment on places where the models don't quite fit with the data. }


To exploit \eref[eq7] we used a set of simple repression constructs based on the \textit{lacUV5} promoter with a single binding site for LacI. The operators were placed at the native wild-type position of the O1 binding site. Each construct was chromosomally integrated and bore either Oid, O1, O2, or O3 as an operator, controlling the expression of yellow fluorescent protein (YFP). The copy number of LacI in each strain was modified using the ribosomal binding site to generate a set of strains with mean copy numbers of 10, 30, 62, 130, 610, and 870 \cite{Garcia2011}. Flow cytometry was used to measure the fold-change in gene expression of our constructs as the concentration of IPTG was titrated between 0 and 5000 $\mu M$. In addition a subset of these strains were also examined by single cell microscopy, particularly those expected to be at the limit of sensitivity by the flow cytometer.

\subsection*{Determination of the \textit{in vivo} MWC Parameters.}

The binding energies $\Delta \varepsilon_{RD,A}$ and LacI copy number have
previously been measured using either single-cell microscopy or bulk measurement \cite{Oehler1994,Vilar2003,Garcia2011, Brewster2014}. This leaves us with three free parameters in \eref[eq7] that are unknown. These are $K_I$, $K_A$, and $\Delta\varepsilon_{IA}$, which reflect the binding affinity of inducer to LacI in the active conformation, the binding affinity of inducer to LacI in the inactive conformation, and the free energy difference between these two conformations. Using previously generated data from our group we found a best estimate of $\beta\Delta\varepsilon_{IA}$ to be equal to 4.5 (See Appendix \ref{AppendixSloppiness}). We applied Bayesian Markov chain Monte Carlo parameter estimation to obtain $K_I$ and $K_A$. Since these parameters are dependent on the properties of LacI and its interaction with inducer rather than the specific regulatory context considered, we can use expression data from any of our strains to estimate $K_I$ and $K_A$. While we could also estimate these parameters with a global fit of all our data, this approach allows us to then \textit{predict} expression across our other strains without any further fitting. Taking this approach, we fit these parameters using our simple repression strain with $R=130$ and an O2 operator. The measured fold-changes in expression are shown in \fref[fig_result1] as a function of inducer concentration. Plotted along with our data is \eref[eq7] using our fitted values for $K_I$ and $K_A$.

\begin{figure}[h]
	\centering \includegraphics[scale=0.5]{extra_figures/fig_fit_explanation_03.pdf}
	\caption{{\bf Fold-change measurements and best fit from an IPTG titration using a simple repression construct containing 130 repressors per cell and O2 operator.} Fold change of RBS1027 O2 simple repression construct as a function of IPTG concentration. The solid points correspond to our experimental data, where error bars in fold-change measurements referring to the SEM ($n=10$). The solid	lines correspond to \eref[eq7] using the parameter estimates of $K_I$ and $K_A$. Values for LacI copy number and operator binding energy are from \cite{Garcia2011}.  The shaded region on the curve represents the uncertainty from our parameter estimates and reflect the 95\% highest probability density region of the parameter predictions for $K_I$ and $K_A$. \talComment{We should either explicitly state the parameter values here or have a table with them in the text (I vote for the former)}} \label{fig_result1}
\end{figure}

\subsection*{MWC Model Predicts Expression as Operator Binding Energy and Repressor Copy Number is Varied.}

Using the parameters we found for $K_I$ and $K_A$ we then predicted the fold-change as a function of IPTG concentration for our 23 other strains with different operator and repressor copy number. \fref[fig_result2] shows the measured fold-change plotted along with these parameter-free predictions. We find excellent agreement between the theoretical predictions and the experimental measurements for these other strains, demonstrating that we have quantitative control over the knobs in our system. Of course, we could have taken this approach with any of our strains and consider a more extensive analysis between parameter estimates of the different datasets in the supplemental information (see Appendix \ref{AppendixParamEstimation}). This agreement between our data and predictions demonstrate the predictive power of our model and suggests we can modify the simple repression motif at will and faithfully predict the induction profile we will observe.

\begin{figure}[h]
	\centering
	\includegraphics[scale=0.5]{fig_theory_vs_data_O2_RBS1027_fit.pdf}
	\caption{{\bf Theoretical fold-change predictions versus experimental measurements using different operator binding energies and repressor copy numbers.} Fold-change measurements for 24 strains are plotted as a function of IPTG concentration. Solid points correspond to our experimental data, where error bars in fold-change measurements referring to the SEM ($n=10$). Solid lines correspond to	\eref[eq7] and use the parameter estimates of $K_I$ and $K_A$ from	strain RBS1027 with an O2 operator. Aside from strain RBS1027 with the O2	operator, the remaining 23 curves reflect parameter-free predictions and demonstrate that the data are consistent with our model. The shaded region on the curve represents the uncertainty from our parameter estimates}
	\label{fig_result2}
\end{figure}

In the above approach we estimated $K_I$ and $K_A$ using fold-change measurements from a single strain and then ask how well our model predicted expression for other simple repression constructs. As an alternative approach, we mentioned that we could have performed a global fit of \eref[eq7] using all of the data plotted in \ref{fig_result2}. We implemented this approach and plot a comparison of the data and theoretical fold-change values in Appendix \ref{AppendixParamEstimation}. These different approaches lead us to essentially comparable results and provide further confidence in our use of \eref[eq7].

\subsection*{Data Collapse Demonstrates a Common Framework for Induction with our MWC Model.}

Finally, while our theory implies a unique response curve for any set of operator and repressor copy number, each seemingly different curve is derived from the a common mechanism. To better see how our model encompasses this, we can rewrite \eref[eq7] in a way that will collapse the predictions for our 24 different strains onto a single master curve. Specifically we write fold-change in the following form,

\begin{equation}\label{eq8}
\foldchange= \frac{1}{1+e^{-\beta F(c)}} ,
\end{equation}
where we call $F(c)$ the Bohr parameter \cite{Phillips2016} that is a more natural scaling variable to plot our fold-change expression data against. Here $F(c)$ is given by

\begin{equation}\label{eq9}
F(c) =  \frac{1}{\beta} log p_A(c)
- \Delta\varepsilon_{RD,A}
log \frac{2R}{N_{\text{NS}}} .
\end{equation}
From \eref[eq7] we can rewrite this as

\begin{equation}\label{eq10}
F(c) = \frac{1}{\beta} log \frac{\left(1+\frac{c}{K_A}\right)^2}{\left(1+\frac{c}{K_A}\right)^2+e^{-\beta  \varepsilon }\left(1+\frac{c}{K_I}\right)^2} - \Delta\varepsilon_{RD,A} log \frac{2R}{N_{\text{NS}}} .
\end{equation}
In \fref[fig_result3] we plot experimental fold-change data against their
associated Bohr parameter and find that the data for all 24 strains collapses onto this single curve. This collapse of our data onto this curve further emphasizes that our model is able to capture the behavior of our simple repression motif across a wide range of thermodynamic parameters. This would not be possible, for example, by approximating these sigmoidal response curves with a commonly used Hill function.

\begin{figure}[h]
	\centering \includegraphics[scale=0.5]{fig_data_collapse_O2_RBS1027_fit.pdf}
	\caption{{\bf Fold-change data from 24 different strains plotted as a function of their Bohr parameter, \eref[eq10].} Solid points correspond to our	experimental data, where error bars in fold-change measurements referring to the SEM ($n=10$). Green points refer to strains containing the O1 operator, blue points refer to strains containing the O2 operator, and red points refer	to strains containing the O3 operator.
		\talComment{(1) Maximum y-range can be capped at 1.1 or less; (2) Legend should
			be reordered; (3) This figure would be much more impressive if we used shades of green/blue/red for the different strains, so that people can see that it is 24 data sets. Frankly, since we have room, we could potentially list all 24 of the strains in the legend}} \label{fig_result3}
\end{figure}
