%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Experimental Design}

To exploit the expression for fold-change \eref[eq7] we used a set of simple
repression \textit{E. coli} constructs with variable repressor copy number $R$
and operator binding energies $\Delta\varepsilon_{RA}$
(\fref[figpolymeraseRepressorStates]\letter{B}) \robComment{Call out knobs, call
	out Figure 1 theory - you did the theory of how all of this works first, before
	the experiments!}.  The copy number of LacI in each strain was modified using
the ribosomal binding site to generate strains with mean LacI dimer copy numbers
of 22, 60, 124, 260, 1220, and 1740 \cite{Garcia2011}. Gene expression was
measured using a Yellow Fluorescent Protein (YFP) gene regulated by a modified
\textit{lacUV5} promoter. In this promoter, operators of different repressor
binding energies were placed at the YFP transcription start site. Each construct
was chromosomally integrated and expression was quantified using Flow Cytometry
as the concentration of IPTG was titrated between 0 and $5\times
10^{-6}\,\,\text{M}$. In addition a subset of these strains were also examined
by single cell microscopy, particularly those expected to be at the limit of
sensitivity of the flow cytometer.

\subsection*{Determination of the \textit{in vivo} MWC Parameters}

The binding energies and LacI copy number have previously been measured using
either single-cell microscopy or bulk measurement \cite{Garcia2011,
	Brewster2014}. This leaves three free parameters in the fold-change \eref[eq7]
that are unknown, $K_I$, $K_A$, and $\Delta\varepsilon_{AI}$, which reflect the
binding affinity of inducer to LacI in the inactive conformation, the binding
affinity of inducer to LacI in the active conformation, and the free energy
difference between these two conformations. From previous
data\cite{Brewster2014}, we infer that $\Delta\varepsilon_{AI} = 4.5k_BT$ (see
Appendix \ref{AppendixSloppiness}) \robComment{I am not entirely clear on what
	this strategy is}. Since these two parameters are dependent on the properties of
LacI and its interaction with the inducer rather than the specific regulatory
context considered, we applied Bayesian parameter estimation using Markov Chain
Monte Carlo to obtain $K_I$ and $K_A$ using fold-change data from a single
strain and then used those to predict expression for the remaining 23 strains
with no further fitting.

\fref[fig_result1] shows how we use fold-change measurements from a single
strain with $R=260$ and an O2 operator ($\Delta\varepsilon_{RA}=-13.9k_BT$,
orange open diamonds), and infer the most likely parameter values for
$K_I=0.56^{+0.05}_{-0.04} \times 10^{-6}\,\, \text{M}$ and $K_A=141^{+28}_{-23}
\times 10^{-6} \,\, \text{M}$ \robComment{explain to me this notation} where the
error represents the 95\% confidence interval. Using these parameters, we
calculated the fold-change \eref[eq7] and the associated confident interval
(orange line \fref[fig_result1]). With all parameters for fold-change
determined, we can predict titration curves for any LacI copy number. In
\fref[fig_result1] we plot the predictions for five other strains \robComment{I think you should show all the predictions, and then make the last sentence ``for any LacI copy number and any choice of operator'' \talComment{but we should ensure that the build up is conceptually clear}} with copy
number spanning three orders of magnitude.

\begin{figure}[h]
	\centering
	\includegraphics[scale=0.5]{extra_figures/fig_fit_explanation_03.pdf}
	\caption{{\bf Using a single strain with $\boldsymbol{R=260}$ predicts the
			fold-change at any other repressor copy number. \robComment{Clunky title. How about ``Predictions for fold-change as a function of repressor copy number''}} IPTG titration of the O2 strain with $R=260$ and
		$\Delta\varepsilon_{RA} = -13.9k_BT$ (orange open circles) can be used to
		determine the thermodynamic parameters $K_I=0.5 \times 10^{-6} \,\, \text{M}$,
		$K_A=141 \times 10^{-6} \,\, \text{M}$, and $\Delta \varepsilon_{AI}=4.5k_BT$
		The solid lines correspond to predicted fold-change for various repressor copy numbers given
		by \eref[eq7]. Error bars of experimental data show SEM ($n=10$) and shaded
		regions \robComment{I need a tutorial on this} denote the 95\% confidence intervals. \robComment{Need to alert
			readers somewhere to the factor of 2 in $R$ values in case they know HG
			results} \talComment{The size of this figure should match the size of \fref[fig_result2] figures.=}} \label{fig_result1}
\end{figure}

\subsection*{Comparison of Experimental Measurements with Theoretical Predictions across Different LacI Copy Numbers and Operator Binding Energies}

In addition to tuning LacI copy number, we can also vary the operator binding
energy. \fref[fig_result2] shows the parameter-free predictions along with
experimental measurements of fold-change for the operators O1, O2, and O3 with
operator binding energies of $-15.3, -13.9, -9.7k_BT$, respectively. Each data
point reflects the mean and SEM from ten independent measurements, where each
measurement represents the mean fluorescence of 40,000 cells.

Across all of our strains with different operators and LacI copy numbers we find
very good agreement between theory and experiment ($R^2=\nathanComment{???}.$).
It is of particular importance to note that the experimental data agrees
remarkably well with several theoretical predictions. For example, all the O1
operators go to zero fold-change in the absence of inducer, signifying that the
repressor binds very strongly to the operator and inhibits gene expression
almost entirely in this regime. The O3 operator exhibits the other limit of weak
binding, and consequently all strains have fold-change approaching one in the
saturating inducer limit, representing that all LacI are inactive and not
inhibiting gene expression. The O2 operator shows an intermediate trend between
these two operators with a small spread of fold-change values both in the
presence and absence of inducer. Another common trend seen in the data is that
all 24 strains transition from their minimum to maximum fold-change between
$10^{-5}$ to $10^{-3} \,\, \text{M}$ IPTG, without any significant shift left or
right. \robComment{Good paragraph - focus on building intuition. With respect to
	fitting, I feel like finally we are getting to the point of saying
	theory-experiment dialogue can ask about 20\% effects. Is this interesting and
	useful?}

\robComment{I think you should have a paragraph on leakiness, dynamic range,
	$[EC_{50}]$, and effective Hill coefficient (referencing Peter Swain's paper).
	Then discuss intuition. \talComment{We need to think hard about how to do this.
		The derivations themselves will be long and technical, and we need to consider
		eLife audience. My gut instinct right now is that we could have a \textit{short}
		paper on the mathematics of this system which would discuss these 4 properties
		as well as what happens when $\Delta\varepsilon_{AI} < 0$ or
		$\Delta\varepsilon_{AI} > 0$, but aimed at a different audience of theorists}}

Another interesting aspect of the theoretical predictions is the width of the
confidence intervals, which increases with repressor copy number and inducer
concentration. The four strains with lowest copy number, including the wild type
strain $R=22$, have tightly constrained fold-change predictions, which implies
that changing any of the model parameters by 10\% negligibly changes the
fold-change. The confidence intervals for all O3 operator strains are tightly
constrained even for those with high repressor copy number. This is likely due
to the fact that this operator binding energy is weak, resulting in moderately
high gene expression even in the presence of more than 1000 repressors
\griffinComment{right?}\nathanComment{???}.

\begin{figure}[h!]
	\centering \includegraphics[scale=0.5]{fig_theory_vs_data_O2_RBS1027_fit.pdf}
	\caption{{\bf Theoretical fold-change predictions versus experimental
			measurements using different operator binding energies and repressor copy
			numbers.} \robComment{Big question: We need to set the standard for the field
			on how to think about theory-experiment comparison. We can't just say the ``fit
			is excellent.'' We need to have some deeper thoughts including what can we do
			next to break the model? Answer 1: Mutate protein, i.e. allude to the next
			paper} Using the O2 strain with $R=260$ (orange open circles) predicts the IPTG
		titration data for all other O2 strains as in \fref[fig_result1]. By changing
		the operator binding energy $\Delta \varepsilon_{RA}$, we can predict the
		titration curves for all \letterParen{A} O1, \letterParen{B} O2, and
		\letterParen{C} O3 strains. Error bars of experimental data show SEM ($n=10$)
		and shaded regions denote the 95\% confidence intervals. \talComment{At the
			risk of being overly picky, I think the ``O1'', ``O2'', ``O3'' labels would
			look better if they were centered inside the grid line rectangles (i.e.
			centered at $(10^{-2.5}, 0.1)$)}} \label{fig_result2}
\end{figure}

We note that rather than using $R=260$ for O2, we could have used any of the
strains to estimate $K_I$ and $K_A$ \robComment{good}, and we consider all such
possibilities in Appendix \ref{AppendixParamEstimation}. As an alternative
approach, we could also perform a global fit using data from all 24 strains to
obtain the best estimate of $K_I$ and $K_A$ for the LacI system (Appendix
\ref{AppendixParamEstimation}). The close agreement between each of these
methods demonstrates our quantitative understanding of induction in the simple
repression architecture.

\subsection*{Data Collapse of 24 Strains onto One Master Curve}

One beautiful aspect of the MWC model is that it offers a unifying perspective
with which to understand all 24 seemingly different induction profiles. To
demonstrate this, we rewrite \eref[eq7] in the following form,
\begin{equation}\label{eq8}
\foldchange= \frac{1}{1+e^{-\beta F(c)}} ,
\end{equation}
where we call $F(c)$ the Bohr parameter \cite{Phillips2015a}, with $F(c)$ given
by \talComment{I changed the exponents of 2 in this equation to $n$ in line with
	the changes in \eref[eq7]. If we keep this, then we have to mention $n=2$ in
	\fref[fig_result1]}
\begin{equation}\label{eq10}
F(c) = - k_BT \left( \log \frac{\left(1+\frac{c}{K_A}\right)^n}{\left(1+\frac{c}{K_A}\right)^n+e^{-\beta  \Delta\varepsilon_{AI} }\left(1+\frac{c}{K_I}\right)^n} + \log \frac{R}{N_{\text{NS}}} e^{- \beta \Delta\varepsilon_{RA}} \right).
\end{equation}
\fref[fig_result3] shows the experimental fold-change data as a function of the
Bohr parameter, with all 198 data points from 24 strains collapsing onto a
single master curve. This collapse emphasizes that the seemingly different
response curves in \fref[fig_result2] all arise from a unified model
\robComment{I would emphasize what the cell cares about is the Bohr parameter.
	There are many ways to get the same fold-change: you could tune $R$, $\Delta
	\varepsilon_{RA}$, or $c$. The system does not care which combination you use,
	as long as you get to the right Bohr parameter value. This is the INTUITION to
	emphasize. \talComment{Building off this point, can add something like:
		``Various biological systems have shown that convergent evolution can emerge by
		tuning different biological parameters \cite{Stern2009}. Indeed, in Appendix
		\ref{AppendixSloppiness}, we discuss how the $K_A$, $K_I$, and $\Delta
		\varepsilon_{AI}$ parameters are not completely independent.''}} with a single
set of thermodynamic parameters.

\begin{figure}[h!]
	\centering \includegraphics[scale=0.4]{fig_data_collapse_O2_RBS1027_fit.pdf}
	\caption{{\bf Fold-change data from 24 different strains collapse onto a single
			master curve.} Experimental data from \fref[fig_result2] is plotted as a
		function of the Bohr parameter \eref[eq10]. \robComment{Larger fonts on all
			plot labels}} \label{fig_result3}
\end{figure}
