\pagebreak
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Results}

\subsection*{The Monod-Wyman-Changeux (MWC) Model \talComment{Tal's Section!}}
\griffinComment{Is the SI a better place for this table?}
\begin{table}
	\small
	\centering \caption{{\bf Key model parameters for induction of an allosteric repressor.} \talComment{We can decide whether we like this, and whether it should be explicitly referenced anywhere in the text}}
	\begin{tabular}{@{\vrule height 10.5pt depth4pt  width0pt}cl}
		\textbf{Parameter} & \multicolumn{1}{c}{\textbf{Description}} \cr
		\hline \noalign{\vskip 2pt}
		\hline
		$K_A, K_I$ & Dissociation constant between the inducer IPTG and the Lac repressor in the active/inactive state \cr
		\hline
		$\Delta \varepsilon_{IA}$ & Difference in free energy between the active and inactive allosteric states of Lac repressor \cr
		\hline
		$\Delta\varepsilon_{P}$ & Binding energy between the Lac promoter and RNAP \cr
		\hline
		$\Delta\varepsilon_{RA}, \Delta\epsilon_{RI}$ & Binding energy between the Lac promoter and the active/inactive Lac repressor \cr
		\hline
		$P$ & Number of RNAP \cr
		\hline
		$R_A, R_I, R$ & Number of active/inactive/total Lac repressor dimers \cr
		\hline
		$p_{\text{bound}}$ & Fraction of time RNAP is bound to Lac promoter, proportional to gene expression \cr
		\hline
		$\foldchange$ & Ratio of gene expression in the presence of Lac repressor to gene expression in the absence of Lac repressor \cr
		\hline
		$N_{NS}$ & The number of non-specific binding sites for the Lac repressor on the \textit{E. coli} genome \cr
		\hline
		$\beta = \frac{1}{k_B T}$ & The inverse product of Boltzmann's constant $k_B$ and the temperature $T$ of the system \cr
	\end{tabular}
	\label{table1}
\end{table}

This paper builds upon an extensive dialogue between theory and experiment in
transcriptional regulation. A first round of experiments measured the
dependence of gene expression on repressor copy number and binding strength
\cite{Garcia2011}. This predictive framework was expanded upon by highlighting the fact that different genes compete for the
same regulatory apparatus. These predictions were shown to be
consistent with their measured counterparts, even permitting the collapse of
data from multiple experiments onto master curves \cite{Brewster2014,
Weinert2014}. In this work, we consider yet another layer of complexity
by predicting changes in gene expression due to an extracellular chemical
inducer.

The ability of the the Lac repressor to regulate transcription has been
previously characterized by an equilibrium model where the probability of each
state of repressor and RNA polymerase promoter occupancy is dictated by the
Boltzmann distribution \cite{Daber2011a, Phillips2015a}. We begin with a
summary of this model. Suppose there are \(P\) RNA polymerases (RNAP) and \(R\)
repressors in a cell. \(R_A\) repressors will be in the active state (the
favored state when the repressor is not bound to an inducer; in this state the
repressor binds tightly to the DNA) and the remaining \(R_I\) repressors will
be in the inactive state (the predominant state when repressor is bound to an
inducer; in this state the repressor binds weakly to the DNA) so that
\(R_A+R_I=R\). Repressors fluctuate between these two conformations in thermodynamic equilibrium\cite{Kern2003}. In this work, we consider a Lac repressor complex composed of two independently operating dimers. Thus, $R$ represents the number of repressor dimers in the cell.

We first model the interaction between the Lac repressor and the DNA by
enumerating all possible states and their corresponding statistical weights. We
consider a simple repression genetic architecture where binding of the
repressor occludes binding of the RNAP. As shown in
\fref[figpolymeraseRepressorStates], the Lac promoter can either be empty,
occupied by RNAP, or occupied by either an active or inactive repressor
molecule. Following the MWC model of allostery, we assign the Lac repressor a
different DNA binding affinity in the active and inactive state. Assume that
there are $N_{NS}$ non-specific sites on the DNA outside the Lac operator where
RNAP or the Lac repressor can bind. \(\Delta\varepsilon_{P}\) represents the
energy difference between RNAP bound to the Lac promoter or bound elsewhere on
the DNA; \(\Delta\varepsilon_{RA}\) and \(\Delta\varepsilon_{RI}\) equal the
difference in energy when the Lac repressor is bound to the Lac operator
compared to when it is bound non-specifically elsewhere on the DNA in the
active and inactive state, respectively. $\beta = \frac{1}{k_BT}$ where $k_B$
is Boltzmann's constant and $T$ is the temperature of the system.

\begin{figure}[h]
	\centering \includegraphics[scale=0.75]{simple_states_and_weights_allosteric_independence.pdf}
	\caption{{\bf States and weights for simple repression. \talComment{Replace with Griffin Figure. Make sure notation is changed to the new $\varepsilon_P$, $\varepsilon_{RA}$, and $\varepsilon_{RI}$. Ensure $N_{NS}$ has italicized subscripts}} Both RNAP (light blue)
		and repressor compete for DNA binding. There are $R_A$ repressors in the active
		state (green) and $R_I$ repressors in the inactive state (red), with the latter
		type typically bound to inducer (gold). The difference in energy between a
		repressor bound to the Lac operator and to another non-specific site on the DNA
		equals $\Delta\epsilon_{RA}$ in the active state and $\Delta\epsilon_{RI}$
		in the inactive state; the $P$ RNAP have a corresponding energy difference
		$\Delta\epsilon_{P}$. The number of active repressors $R_A$ includes
		repressors that are unbound, singly bound, or doubly bound to inducer, although
		the majority of active state repressors will not be bound to inducer (which
		pushes them into the inactive state). Similarly, the $R_I$ term includes all
		inactive state repressors bound to any number of inducer molecules, with the
		most prevalent state shown in the figure. The factor of 2 in the repressor
		weights comes from the ability of either repressor dimer to bind to DNA.}
	\label{figpolymeraseRepressorStates}
\end{figure}



In thermodynamic models of transcription, gene expression is proportional to the probability $p_{\text{bound}}$ that the RNAP is bound to the Lac promoter which is given by
\begin{equation}\label{eq2}
p_{\text{bound}}(R_A, R_I \mid R)=\frac{\frac{P}{N_{NS}}e^{-\beta  \Delta\varepsilon_{P}}}{1+\frac{R_A}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}}+\frac{R_I}{N_{NS}}e^{-\beta  \Delta\varepsilon_{RI}}+\frac{P}{N_{NS}}e^{-\beta\Delta\varepsilon_{P}}}.
\end{equation}
Measuring $p_{\text{bound}}$ directly is fraught with experimental difficulties
\cite{Bintu2005}. Instead, we measure the fold-change in gene expression due the presence of the repressor. We define fold-change as the ratio of gene
expression relative to that in the absence of repressor.
\begin{equation}\label{eq3}
\foldchange\equiv \frac{p_{\text{bound}}(R_A, R_I \mid R)}{p_{\text{bound}}(R = 0)}.
\end{equation}
We can simplify this expression using two approximations:
$\frac{P}{N_{NS}}e^{-\beta\Delta\varepsilon_{P}}\ll 1$ and
$\frac{R_A}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \ll \frac{R_I}{N_{NS}}e^{-\beta \Delta\varepsilon_{RI}}$. The former is the weak promoter
approximation, which assumes that binding of the RNAP is too weak to compete with
repressor binding\cite{Brewster2012}.
The latter states that the inactive repressor binds weakly to the
DNA compared to an active repressor. Using these
approximations, the fold-change reduces to the form
\begin{equation}\label{eq4}
\foldchange \approx \left(1+\frac{R_A}{N_{NS}}e^{-\beta  \Delta\varepsilon_{RA}}\right)^{-1}.
\end{equation}

We now introduce the role of inducer binding, which changes the number of
repressors in the active and inactive allosteric states. We begin by defining
$p_A(c) \equiv \frac{R_A(c)}{R}$ to be the fraction of repressors in the
active state given a concentration $c$ of the gratuitous inducer IPTG, so that
\begin{equation}
\foldchange = \left( 1+p_A(c) \frac{R}{N_{NS}}e^{-\beta
	\Delta\varepsilon_{RA}} \right)^{-1}. \label{eq5}
\end{equation}

%The Lac repressor is a tetrameric protein, a dimer of dimers, with four
%identical binding sites for an inducer. It is unknown whether the two dimers are
%allosterically independent (i.e. whether one dimer can be active while the other
%is inactive) or allosterically linked (with both dimers being either
%simultaneously active or simultaneously inactive). While no direct measurements
%have as of yet been carried out to definitively distinguish between these two
%models, in Appendix \ref{AppendixAllostery} we describe a simple experiment
%which can do precisely this. In this paper, we proceed with the assumption that
%the two repressor dimers are allosterically independent.
%%\stephanieComment{For now I think this is fine--but if our experiments suggest otherwise when we get more data, I think we should be willing to change our model}

\begin{figure}[h]
	\centering \includegraphics[scale=0.2]{inducer_states_and_weights.pdf}
	\caption{{\bf The eight states of a Lac repressor dimer. \talComment{Unify color scheme, clarify that the bottom is the sum of the column}} The Lac repressor ha\globalScalePlotss
		an active conformation (green, left column) and inactive conformation (red,
		right column), with the energy difference between these two states given by
		$\Delta \varepsilon_{IA}$. In each conformation, the repressor can bind an inducer (gold) at
		two sites. Each state is shown with its corresponding Boltzmann weight. The top
		dimer can be in the active or inactive state independent of the bottom dimer.
		The top dimer is drawn partly opaque because its different states will not
		effect the probability that the bottom dimer is active. %		If the sum of the
		% active state weights shown (bottom left) is greater than the sum of
		%		the inactive state weights (bottom right), the repressor is more likely to be
		%		in the active state.
	} \label{figrepressorInducerStates}
\end{figure}

As shown in \fref[figrepressorInducerStates], we can enumerate the relative
likelihood of the eight possible conformations of a repressor (each dimer
can be in an active or inactive state, and its two inducer binding sites
can be empty or occupied), using the difference in energy $\Delta \varepsilon_{AI}$ between a
repressor in the active and inactive state. From these eight states, we can
compute the probability $p_A(c)$ that a repressor will be in the active state
as the sum of the weights of the active states divided by the sum of the weights
of all possible states, namely,
\begin{equation}\label{eq6}
p_A(c)=\frac{\left(1+\frac{c}{K_A}\right)^2}{\left(1+\frac{c}{K_A}\right)^2+e^{-\beta  \Delta \varepsilon_{AI} }\left(1+\frac{c}{K_I}\right)^2}.
\end{equation}
Substituting this result into \eref[eq5] yields the
complete formula
\begin{equation}\label{eq7}
\foldchange= \left(
1+\frac{\left(1+\frac{c}{K_A}\right)^2}{\left(1+\frac{c}{K_A}\right)^2+e^{-\beta  \Delta \varepsilon_{AI} }\left(1+\frac{c}{K_I}\right)^2}\frac{R}{N_{NS}}e^{-\beta \Delta\varepsilon_{RA}} \right)^{-1}.
\end{equation}

In this work, we titrate the concentration \(c\) of the inducer IPTG using
\textit{E. coli} strains with experimentally determined values of $R$ and
$\Delta\varepsilon_{RA}$ \cite{Garcia2011}. Approximating the number of
nonspecific binding sites as the length of the \textit{E. coli} genome $N_{NS} =
4.6 \times 10^6$, the
fold-change in gene expression will depend solely on 3 parameters: the free
energy between the active and inactive states of the repressor
($\Delta\varepsilon_{AI}$) and the inducer dissociation constants for the
repressor in the active state ($K_A$) and inactive state ($K_I$).
