%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Introduction}

%paragraph 1: the ubiquitous induction phenomenon -  a key part of the
%central dogma story that is the meeting point of cell signaling and
%gene regulation.  I would also touch on the “Whence Cometh the Allosterome”
%paper from PNAS that notes how hard it is to do any sort of high
%throughput analysis of allostery.

The communication of organisms with the external world is one of the most ubiquitous phenomena in biology. At the cellular level, this communication takes the form of signaling pathways. One of the most pervasive categories of cell signaling is allosteric regulation, which affects gene expression when transcription factors have their activity altered by the binding of a ligand. Despite the overarching importance of this model of signaling, we have lacked a rigorous dialogue between theory and experiment, in part due to the difficulty of analyzing allostery in a high-throughput manner \cite{Lindsley2006}. To that end, we explore cell signaling in the context of one of the most widespread bacterial regulatory architectures, the simple repression motif \cite{Rydenfeldt2014}.

%Paragraph 2: The challenge we undertake - a predictive theory of the response
%of genetic circuits to allosteric inputs.  What we require of such a predictive theory
%is not that one can fit some sigmoidal curve, but rather something that allows
%us to understand all aspects ranging from leakiness to the EC50 and effective
%Hill coefficient to the dynamic range.  

Here we propose a predictive theory of the response of genetic circuits to allosteric inputs. Such a theory should allow us to understand all aspects of a ligand titration curve, such as the leakiness, the $EC_{50}$, the slope of the inflection region, and the dynamic range. The inputs to our model must therefore have a strong biophysical basis. A strong theory of allosteric induction is likewise required to place each of these inputs in their proper context. 

%Paragraph 3: WE show a general statistical mechanical theory of the induction
%phenomenon using the famous MWC model.   We show how this model
%can be used for various transcription factors and modes of induction (i.e. how
%many inducer binding sites, whether inducer switches from active to inactive
%or the other way around).

As the foundation for our theory of allosteric induction, we use a statistical mechanical rendering of the Monod-Wyman-Changeux (MWC) model \cite{MONOD1965}. The MWC model stipulates that an allosteric protein is capable of
accessing two distinct conformations -- an active and
inactive state -- in thermodynamic equilibrium. A protein's affinity
for ligand is different for the active and inactive states, such that the
equilibrium between these states shifts with changing concentration of ligand
\cite{Marzen2013}. We show how this model can be used for various transcription factors and modes of induction, accounting for variations such as the number of ligand binding sites, the regulatory effect of the ligand (i.e. whether it switches the transcription factor from active to inactive or vice versa), and cooperativity between transcription factor subunits. \stephanieComment{At this point we don't actually discuss variations to the allosteric properties of the transcription factor, but I like the idea of addressing it by showing theory curves for hypothetical TFs with various properties. This could either be part of our primary results or a cool appendix for the SI.}

%Paragraph 4: we design and implement an experimental system in which
%we tune many distinct parameters in the genetic circuit itself - i.e. the binding
%energies and transcription factor copy numbers.  We pose a simple question.
%Once we have determined the three parameters that characterize
%the transcription factor, can we then use that completely characterized
%transcription factor in all of these different contexts and predict what the outcome
%will be.

To show how our model accounts for the biophysical parameters unique to each individual regulatory motif, we use the well-characterized components of the \textit{lac} regulatory system. We design and implement an experimental system in which we tune several distinct parameters in the genetic circuit itself, such as the transcription factor binding energies and copy numbers. This leaves the allosteric properties of the transcription factor as the only free parameters, which should remain constant in each of our variations on the genetic circuit. We then assess the predictive nature of our model with the following question: once we have determined the three parameters that characterize the allosteric properties of the transcription factor, can we then apply these parameters to each of our designed genetic circuits and determine the regulatory outcome?


%Paragraph 5: We show the measurements, the comparison between the theory
%and the experiment and propose a new fundamental parameter that allows
%us to see how there are many ways to reach the same level of transcription
%and all of these are equivalent.

To answer this question, we measure the fold-change values of each of our genetic circuits over several orders of magnitude of ligand concentration. We show that we can use the ligand response curve of a single genetic circuit to determine the allosteric parameters of the transcription factor, and then use these parameters to accurately predict the response curves for each of the remaining genetic circuits. Moreover, we propose a new fundamental parameter (which we call the Bohr parameter) that captures the regulatory range of each genetic circuit on a single single curve, showing that a broad selection of thermodynamic parameters produce the same regulatory result, meaning that they are all biologically equivalent. 

