%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section*{Introduction}

Biological mechanisms are complex and diverse, and as a consequence our
understanding is often characterized by a slow accumulation of qualitative
information that is often difficult to fit into a unifying framework.
Nevertheless, the last decade has yielded new techniques that have given us
unprecedented access to the fine details underlaying a broad array of
biological phenomena, from the exquisite dissection of the gene regulatory
network of the sea-urchin developmental program [Peters and Davidson] to the
\textit{in vivo} imaging of the central dogma [Ido's MS2, Hernan's MS2, Suntag
paper, Xiaowei recent paper] to the complex task of regulating gene expression
\cite{Garcia2011}. New genetic reporters and advancements in single-cell
measurements have brought about a new level of precision measurements
traditionally only achievable in more quantitative fields such as physics and
chemistry. Such high quality data is now routinely generated in various fields
of biology, and it calls for a similar expansion in our use of theoretical
frameworks to conceptualize and understand the richness of modern biological
data sets. At this level, fine details of a system - the steepness of the
response, the limits of minimum and maximum response, etc. - can now be
resolved to go beyond a general sigmoidal shapes.

Statistical mechanical models of gene regulation have proven to be very
effective not only in predicting the mean expression level as a function of
natural parameters such as the transcription factor copy number and its binding
energy to the DNA \cite{Garcia2011}, but also in quantifying the effects of DNA
looping \cite{Boedicker2013a} and competing binding sites \cite{Brewster2014}.
Here, we extend such models to examine allostery as a mechanism for
transcriptional regulation. Allostery is a pervasive property of biological
molecules. This principle helps explain a large array of phenomena ranging from
quorum sensing and chemotaxis, oxygen binding by hemoglobin, and signal
transduction via GPCRs. Here, we consider the Monod-Wyman-Changeux (MWC) model
\cite{MONOD1965}, which stipulates that an allosteric protein is capable of
accessing two distinct conformations -- an active and
inactive state -- in thermodynamic equilibrium. A protein's affinity
for ligand is different for the active and inactive states, such that the
equilibrium between these states shifts with changing concentration of ligand
\cite{Marzen2013}.

In this work, we focus on the rich interplay between theory and experiment in
transcriptional regulation, where quantitative dissection of input-output
functions over the last decade has significantly pushed our ability to
manipulate and understand such systems. We focus on one of the
most abundant genetic architectures -- the simple repression motif [Mattias
paper on regulonDB] -- to test our predictions. Specifically, we use a simple repression promoter constructed of the
well-characterized components of the \textit{E. coli lac}
operon and the famously allosteric Lac repressor to quantiatively test a statistical mechanical model of induction using the MWC framework.

One remarkable conclusion of this theory is that we can use the response of
a single strain of known repressor copy number and operator binding energy to
make parameter free predictions for the induction profile of any other simple
repression architecture. To test this, we used 20 additional strains in which
repressor copy numbers and the Lac repressor binding energies have been varied,
and measured the induction profile of each. The remarkable agreement between
these 20 predicted sets and the experimental data demonstrates how all of these
induction profiles collapse onto a master curve governed by a set of underlying
thermodynamic parameters. These results show that quantitative models are an incredibly powerful tool for
understanding the myriad ways in which cells establish control over
their expression profiles. While many combinations of parameters are capable of
achieving the same fold-change values, these titration
curves are significantly affected by both repressor copy number and binding
energy, and we predict these effects neatly with our models. One might imagine
extending these models to predict the consequences of other perturbations such
as mutations, allosteric activation, or more complex regulatory architectures.
