\pagebreak
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection*{Experimental Design}

%\stephanieComment{General comments on this section: Mainly, I think we could expand upon the prediction of expression and provide more justification for data	collapse. We might want to refer to some quantitative comparison between the	quality of the fits from a single strain vs. the fits from all the data sets, and comment on places where the models don't quite fit with the data. }


To exploit \eref[eq7] we used a set of simple repression constructs based on the \textit{lacUV5} promoter with a single binding site for LacI. The operators were placed at the native wild-type position of the O1 binding site. Each construct was chromosomally integrated and bore either Oid, O1, O2, or O3 as an operator, controlling the expression of yellow fluorescent protein (YFP). The copy number of LacI in each strain was modified using the ribosomal binding site to generate strains with mean dimer copy numbers of 22, 60, 124, 260, 1220, and 1740 \cite{Garcia2011}. Flow cytometry was used to measure the fold-change in gene expression of our constructs as the concentration of IPTG was titrated between 0 and 5000 $\mu M$. In addition a subset of these strains were also examined by single cell microscopy, particularly those expected to be at the limit of sensitivity by the flow cytometer.

\subsection*{Determination of the \textit{in vivo} MWC Parameters}

The binding energies $\Delta \varepsilon_{RA}$ and LacI copy number have previously been measured using either single-cell microscopy or bulk measurement \cite{Oehler1994,Vilar2003,Garcia2011, Brewster2014}. This leaves us with three free parameters in the fold-change \eref[eq7] that are unknown. These are $K_I$, $K_A$, and $\Delta\varepsilon_{IA}$, which reflect the binding affinity of inducer to LacI in the active conformation, the binding affinity of inducer to LacI in the inactive conformation, and the free energy difference between these two conformations. Previous data suggests $\beta\Delta\varepsilon_{IA} = 4.5$, and we will use this value for the remainder of this paper (see Appendix \ref{AppendixSloppiness}). Since these two parameters are dependent on the properties of LacI and its interaction with inducer rather than the specific regulatory context considered, we can use expression data from any of our strains to estimate $K_I$ and $K_A$. We applied Bayesian Markov chain Monte Carlo parameter estimation to obtain $K_I$ and $K_A$ using fold-change data from a single strain and then use those to \textit{predict} expression for the remaing 23 strains with no further fitting.
%While we could also estimate these parameters with a global fit of all our data, this approach allows us to then \textit{predict} expression across our other strains without any further fitting.

\fref[fig_result1] shows how we can take fold-change measurements from a single strain, $R=260$ with an O2 operator (orange open circles), and extract the parameters for $K_I=0.5 \times 10^{-6} \,\, \text{M}$ and $K_A=141 \times 10^{-6} \,\, \text{M}$. Using these parameters, we then predicted the fold-change \eref[eq7] and the 95\% confident interval (orange line). With all parameters for fold-change  determined, we can predict titration curves for any LacI copy number. In \fref[fig_result1] we plot the predictions for five other strains with copy number spanning three orders of magnitude.

\begin{figure}[h]
	\centering \includegraphics[scale=0.5]{extra_figures/fig_fit_explanation_03.pdf}
	\caption{{\bf Using a single strain with $\boldsymbol{R=260}$ predicts the fold-change at any other repressor copy number.} IPTG titration of the O2 strain with $R=260$ (orange open circles) can be used to determine the thermodynamic parameters $K_I=0.5 \times 10^{-6} \,\, \text{M}$, $K_A=141 \times 10^{-6} \,\, \text{M}$, and $\Delta \varepsilon_{RA}=4.5 k_B T$. The solid lines correspond to fold-change for various LacI copy numbers given by \eref[eq7]. Error bars of experimental data show SEM ($n=10$) and shaded regions denote the 95\% confidence intervals.} \label{fig_result1}
\end{figure}

\subsection*{Comparison of Experimental Measurements with Theoretical Predictions across Different LacI Copy Numbers and Operator Binding Energies}

With the parameters $K_I$, $K_A$, and $\Delta\varepsilon_{IA}$ determined, we can not only predict fold-change for varying LacI copy number, but also across the operators O1, O2, and O3, each with a different repressor binding energy. \fref[fig_result2] shows the parameter-free predictions along with experimental measurements of fold-change for each operator. Each data point reflects the mean and SEM from $n=10$ independent measurements, where each measurement represents the mean fluorescence of 40,000 cells.

Across all of our strains with different operators and LacI copy numbers we find very good agreement between theory and experiment ($R^2=\nathanComment{???}.$). We found the \nathanComment{???}. of the data points to fall within 2 standard deviations of the 95\% confidence interval. In particular it is interesting to note that the theory matches the experimental trends. For example, all the O1 operators ($\Delta\varepsilon_{RA}=-15.3 k_B T$) go to zero fold-change in the absence of inducer, signifying that the repressor binds very strongly to the operator and inhibits gene expression almost entirely in this regime. The O3 operator exhibits the other limit of weak binding ($\Delta\varepsilon_{RA}=-9.7 k_B T$), and consequently all strains have fold-change approaching one in the saturating inducer limit, representing that all LacI are inactive and not inhibiting gene expression. The O2 operator shows an intermediate trend between these two operators with a small spread of fold-change values both in the presence and absence of inducer. Another common trend seen in the data is that all 24 strains transition from their minimum to maximum fold-change between $10^{-5}$ to $10^{-3} \,\, \text{M}$ IPTG, without any significant shift left or right.

Another interesting aspect of the theoretical predictions is the width of the confidence intervals, which increases with repressor copy number. The four strains with lowest copy number, including the wild type strain $R=22$, have tightly constrained fold-change predictions, which implies that changing any of the model parameters by 10\% negligibly changes the fold-change. The confidence intervals for all O3 operator strains are tightly constrained even for those with high repressor copy number, \nathanComment{???}.
%Using the parameters we found for $K_I$ and $K_A$ we then predicted the fold-change as a function of IPTG concentration for our 23 other strains with different operator and repressor copy number. \fref[fig_result2] shows the measured fold-change plotted along with these parameter-free predictions. We find excellent agreement between the theoretical predictions and the experimental measurements for these other strains, demonstrating that we have quantitative control over the knobs in our system.


\begin{figure}[h!]
	\centering
	\includegraphics[scale=0.5]{fig_theory_vs_data_O2_RBS1027_fit.pdf}
	\caption{{\bf Theoretical fold-change predictions versus experimental measurements using different operator binding energies and repressor copy numbers.} Using the O2 strain with $R=260$ (orange open circles) predicts the IPTG titration data for all other O2 strains as in \fref[fig_result1]. By changing the operator binding energy $\Delta \varepsilon_{RA}$, we can predict the titration curves for all \letterParen{A} O1, \letterParen{B} O2, and \letterParen{C} O3 strains. Error bars of experimental data show SEM ($n=10$) and shaded regions denote the 95\% confidence intervals.}
	\label{fig_result2}
\end{figure}

We note that rather than using $R=260$ for O2, we could have used any of the strains to estimate $K_A$ and $K_I$, and we consider all such possibilities in Appendix \ref{AppendixParamEstimation}. As alternative approach, we could also perform a global fit using data from all 24 strains to obtain the best estimate of $K_I$ and $K_A$ for the LacI system (Appendix \ref{AppendixParamEstimation}). The close agreement between each of these methods demonstrates our quantitative understanding of the LacI system and more generally, of transcriptional regulation.

%In the above approach we estimated $K_I$ and $K_A$ using fold-change measurements from a single strain and then ask how well our model predicted expression for other simple repression constructs. As an alternative approach, we mentioned that we could have performed a global fit of \eref[eq7] using all of the data plotted in \ref{fig_result2}. We implemented this approach and plot a comparison of the data and theoretical fold-change values in Appendix \ref{AppendixParamEstimation}. These different approaches lead us to essentially comparable results and provide further confidence in our use of \eref[eq7].

\subsection*{Data Collapse of 24 Strains onto One Master Curve}

One beautiful aspect of the MWC model is that it offers a unifying perspective with which to understand all 24 seemingly different curves. To demonstrate this, we rewrite \eref[eq7] in the following form,
\begin{equation}\label{eq8}
\foldchange= \frac{1}{1+e^{-\beta F(c)}} ,
\end{equation}
where we call $F(c)$ the Bohr parameter \cite{Phillips2016}, with $F(c)$ given by
\begin{equation}\label{eq10}
F(c) = - k_B T \left( \log \frac{\left(1+\frac{c}{K_A}\right)^2}{\left(1+\frac{c}{K_A}\right)^2+e^{-\beta  \varepsilon }\left(1+\frac{c}{K_I}\right)^2} + \log \frac{R}{N_{\text{NS}}} e^{- \beta \Delta\varepsilon_{RA}} \right).
\end{equation}
We call $F(c)$ the Bohr parameter in reference to the work by Christian Bohr who showed that different hemoglobin binding curves at different pH concentrations all collapse onto a single curve as a function of a similar master variable.

\fref[fig_result3] shows the experimental fold-change data as a function of the Bohr parameter, with all 200 data points from 24 strains collapsing onto this single master curve. This figure emphasizes that the seemingly different response curves in \fref[fig_result2] all arise from a unified model with a single set of thermodynamic parameters.

\begin{figure}[h]
	\centering \includegraphics[scale=0.5]{fig_data_collapse_O2_RBS1027_fit.pdf}
	\caption{{\bf Fold-change data from 24 different strains collapse onto a single master curve.} Experimental data from \fref[fig_result2] is plotted as a function of the Bohr parameter \eref[eq10].} \label{fig_result3}
\end{figure}
