---
layout: post
title: About
cover: about.png
categories: posts
---
# An Allosteric Theory of Transcription Factor Induction
This website serves as a record for the experimental and theoretical work described in the publication "An Allosteric Theory For Transcription Factor Induction" by Manuel Razo-Mejia<sup>\*</sup>, Stephanie Barnes<sup>\*</sup>, Nathan Belliveau<sup>\*</sup>, Griffin Chure<sup>\*</sup>, Tal Einav<sup>\*</sup>, and Rob Phillips (<sup>\*</sup>contributed equally).


The paper can be found on the [bioRxiv](http://biorxiv.org/content/early/2017/02/22/111013) and [arXiv](https://arxiv.org/abs/1702.07460).  You can download PDFs of the current version and the supplementary information below:

* [Main Text](http://www.rpgroup.caltech.edu/publications/RazoMejia_et_al_2017.pdf)

* [Supplementary Information](http://www.rpgroup.caltech.edu/publications/RazoMejia_et_al_2017_SI.pdf)


## Abstract

Allosteric molecules serve as regulators of cellular signaling across all
domains of life.  We present a theory of allosteric transcriptional regulation
that permits quantitative predictions for how physiological responses are tuned to environmental stimuli. To test the model's predictive
power, we apply it to the ubiquitous simple repression motif in *Escherichia coli*. We
measure the fold-change in gene expression at different inducer concentrations in a collection of strains that span a range of
repressor copy numbers and operator binding strengths. After inferring the inducer dissociation constants using data
from a single strain, we show the broad reach of the model by predicting the induction profiles of all other strains.
Finally, we derive an expression for the free energy which enables us to collapse all of the data into a single master curve, capturing all the phenomenology of the induction profiles.
